proteus/TwoPhaseFlow/TwoPhaseFlowProblem.py

```diff
@@ -39,10 +42,12 @@ def __init__(self,
                 assert type(nnz)==int, "Provide (int) nnz"
         assert domain is not None, "Provide a domain"
         assert triangleFlag in [0,1,2], "triangleFlag must be 1, 2 or 3"
-        assert type(initialConditions)==dict, "Provide dict of initial conditions"
```

Comment:
If `initialConditions` is `None` I would like the code to break and let me know that I need to provide them. The same is true for the `boundaryConditions`

Reply:
For `boundaryConditions`, I removed this check because if SpatialTools is used, then there is no need to provide `boundaryConditions`. I can modify the assertion to check that it is not `None` or that the domain is using SpatialTools (and therefore it is okay to be `None`)

Reply:
The thing about the `boundaryConditions` is a good idea. So should we put back the assertion of `initialConditions` being not None? The way you have it now it will not break if you don't provide `initialConditions`, right?